### PR TITLE
polymc: Add version 1.0.6

### PR DESCRIPTION
**Pull request by lem6ns**

### CODE REVIEW

File: bucket/polymc.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "version": "1.0.6",
+    "description": "An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability.",
+    "homepage": "https://polymc.github.io/",
+    "license": "GPL-3.0-only",
```
Comment (binex-dsk) [inline bucket/polymc.json:5]: its gpl-3.0-or-later

**Review by binex-dsk — CHANGES_REQUESTED** [review]

File: bucket/polymc.json (hunk 1 above)
Comment (lem6ns) [reply bucket/polymc.json]: Fixed, thank you.

**Comment by UnicorNora** [comment]
i think this can be merged

**Review by Calinou — APPROVED** [review]
Thanks!

**Comment by dada513** [comment]
@lem6ns sorry for contacting you here, but mind updating the package to 1.1.0? Since now it uses github releases instead of the old packages url, the package (and updater) needs to be updated

**Pull request merged**